packages/Webkul/Product/src/Database/Migrations/2024_10_08_180506_add_min_sellable_quantity_to_product.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('products', function (Blueprint $table) {
+            $table->integer('min_sellable_qty')->default(1)->after('additional');
+        });
```

Comment:
I believe we should not use the product table directly to determine the minimum sellable quantity.

Reply:
I added in product table, based on my experience in SAP Commerce and Prestashop. I can change but please suggest me the table in which I should add the field.

Reply:
In Bagisto, inventory-related aspects are managed through the `product_inventories` and `product_inventory_indices` tables. Additionally, the minimum sellable quantity (`min_sellable_qty`) can be configured from these tables as well.

Reply:
Hi @amit-webkul,
this means that if I have more than one inventories, I must replicate the same value for all inventories?

Reply:
@amit-webkul,

Inventory is not responsible for managing the minimum sellable quantity; this responsibility lies with the product. I agree that he added it directly to the products table, but this needs to be changed. It should be handled through the attribute system.

Reply:
Hi @devansh-webkul,
you suggest to use the `attributes` table to manage the field, don't you?

Reply:
Yes.

Reply:
@devansh-webkul 
thanks, I added the new attribute field.